stdlib/format.mli

```diff
@@ -1537,3 +1407,135 @@ val kasprintf : (string -> 'a) -> ('b, formatter, unit, 'a) format4 -> 'b
 
   @since 4.03
 *)
+
+(** {1:examples Format by example}
```

Comment:
We know we are on the `Format` page. `Examples` is fine in my opinion (and more consistent).